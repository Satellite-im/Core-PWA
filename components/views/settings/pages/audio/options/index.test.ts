import { expect } from '@jest/globals'
import {
  Bitrates,
  SampleSizes,
} from '~/components/views/settings/pages/audio/options/index'

describe('check Bitrates constants', () => {
  const inst: any = Bitrates

  it('should return the Bitrates constants', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the Bitrates constants', () => {
    expect(inst).not.toEqual({})
  })
})

describe('check SampleSizes constants', () => {
  const inst: any = SampleSizes

  it('should return the SampleSizes constants', () => {
    expect(inst).toMatchSnapshot()
  })

  it('should not return the SampleSizes constants', () => {
    expect(inst).not.toEqual({})
  })
})
