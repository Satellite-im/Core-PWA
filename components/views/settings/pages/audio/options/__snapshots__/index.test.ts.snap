// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`check Bitrates constants should return the Bitrates constants 1`] = `
Array [
  Object {
    "text": "8kbps",
    "value": 8000,
  },
  Object {
    "text": "64kbps",
    "value": 64000,
  },
  Object {
    "text": "96kbps",
    "value": 96000,
  },
  Object {
    "text": "256kbps",
    "value": 256000,
  },
  Object {
    "text": "320kbps",
    "value": 320000,
  },
  Object {
    "text": "510kbps",
    "value": 510000,
  },
  Object {
    "text": "1411kbps",
    "value": 1411000,
  },
]
`;

exports[`check SampleSizes constants should return the SampleSizes constants 1`] = `
Array [
  Object {
    "text": "2bits",
    "value": 2,
  },
  Object {
    "text": "8bits",
    "value": 8,
  },
  Object {
    "text": "24bits",
    "value": 24,
  },
  Object {
    "text": "32bits",
    "value": 32,
  },
]
`;
